parlai/scripts/eval_model.py

```diff
@@ -44,12 +44,14 @@ def setup_args(parser=None):
         'number of tasks.',
     )
     parser.add_argument(
+        '-mcs',
         '--metrics',
         type=str,
-        default='all',
-        help='list of metrics to show/compute, e.g. '
-        'ppl, f1, accuracy, hits@1.'
-        'If `all` is specified [default] all are shown.',
+        default='default',
+        help='list of metrics to show/compute, e.g. all, default,'
+        'or give a list split by , like '
+        'ppl, f1, accuracy, hits@1, rouge, bleu'
```

Comment:
I think you need no spaces, right?

Reply:
Sorry I don't understand, what do you mean by space here?

Reply:
I'm genuinely surprised it's `--metrics 'ppl, f1, accuracy'` instead of  `--metrics 'ppl,f1,accuracy'`. Where do you handle that?

Reply:
aha, sorry I didn't consider that....I was just brainlessly following the coding habits.